templates/gallery/partials/tagline.snip.html

```diff
@@ -0,0 +1,21 @@
+{{!
+<!---
+Copyright 2017 The AMP Start Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+}}
+
+<div class="gallery-component-tagline">
+  <div>Gallery of Modern Art</div>
```

Comment:
Google will _probably_ want this to be read from a variable. I know they'll ask use to read from `"heading": "Gallery of Modern&nbsp;Art",`, but the non breaking space will probably make that impossible, so it should be fine as its own new variable in that JSON file.